tests/subsys/net/lib/aws_fota/aws_fota_json/src/main.c

```diff
@@ -20,16 +20,53 @@ static void test_notify_next(void)
 	char job_id[100];
 	char hostname[100];
 	char file_path[1000];
+	int version_number;
 
 	/* Memset to ensure correct null-termination */
 	memset(job_id, 0xff, sizeof(job_id));
 	memset(hostname, 0xff, sizeof(hostname));
 	memset(file_path, 0xff, sizeof(file_path));
 
-	ret = aws_fota_parse_notify_next_document(encoded, sizeof(encoded) - 1,
-			job_id, hostname, file_path);
+	ret = aws_fota_parse_DescribeJobExecution_rsp(encoded,
+						      sizeof(encoded) - 1,
+						      job_id, hostname,
+						      file_path,
+						      &version_number);
 
+	zassert_equal(ret, 1, NULL);
 	zassert_true(!strcmp(job_id, expected_job_id), NULL);
+	zassert_equal(version_number, 1, NULL);
+	zassert_true(!strcmp(hostname, expected_hostname), NULL);
+	zassert_true(!strcmp(file_path, expected_file_path), NULL);
+}
+
+static void test_parse_job_execution_missing_field(void)
```

Comment:
Add to test suite

Reply:
It's currently not working so I'll leave it out so that CI won't complain until it's fixed in a separate PR.